src/main/java/uk/gov/hmcts/reform/cpo/domain/CasePaymentOrder.java

```diff
@@ -12,6 +13,7 @@
 
     private UUID id;
 
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-ddTHH:mm:ssZ")
     private LocalDateTime createdTimestamp;
 
     private LocalDateTime effectiveFrom;
```

Comment:
Is the same formatting required on `effectiveFrom`?

Reply:
👍